sdk/servicebus/service-bus/package.json

```diff
@@ -93,6 +93,9 @@
     "requiredResources": {
       "Azure Service Bus": "https://docs.microsoft.com/azure/service-bus-messaging"
     },
+    "dependencyOverrides": {
```

Comment:
In Container Registry the `dependencyOverrides` was under `requiredResources` while here the two are siblings...

Reply:
Yes they should be siblings. I updated container registry package.json.

Reply:
Does this just override the rush config? I'm curious why service-bus needs it but not event hubs.

Reply:
This is for the serviceBus samples though